main.go

```diff
@@ -121,7 +125,9 @@ func main() {
 	flags.BoolVar(&opt.SkipBugLookup, "skip-bug-lookup", opt.SkipBugLookup, "Do not attempt to find bugs that match test/job failures")
 	flags.StringVar(&opt.LogLevel, "log-level", defaultLogLevel, "Log level (trace,debug,info,warn,error)")
 	flags.BoolVar(&opt.LoadTestgrid, "load-testgrid", true, "Fetch job and job run data from testgrid")
+
 	flags.BoolVar(&opt.LoadProw, "load-prow", opt.LoadProw, "Fetch job and job run data from prow")
+	flags.StringVar(&opt.Config, "config", opt.Config, "Configuration for Prow Sippy")
```

Comment:
Anything more we can say here, is it optional/required, what it should contain, and do we want to link it to prow sippy? Seems like it might be generally useful in future but I'm not sure.

Any particular reason to use a directory with assumed filenames vs cli-arg per config file?

Reply:
So if mode is ocp it looks for the config file of a known name? I kind of prefer it to be explicit.  Imagine having to do something like the sdn/ovn comparison again, it's easy to customize a list of jobs.

Reply:
Yea sorry it's just one config file.  

The generator can take a customization file to override the generated config. There's some docs in [config/README.md](https://github.com/openshift/sippy/pull/476/files#diff-6e8c7fdec58c9cb55af12d91063af268d43c753cf477bc41ec4e938f1b429b6f), but happy to make it clearer or refactor that if it's not clear